### DISCUSSION TITLE
Section 4: Lesson 7 - Random Numbers

### DISCUSSION THREAD

**Discussion by LordAdhaar** [Cyfrin/foundry-full-course-cu#2673] [Q&A]
<img width="1470" alt="Screenshot 2024-10-13 at 8 41 57 PM" src="https://github.com/user-attachments/assets/bcb5bfa6-bb7a-4d5a-9d10-b48d33e496d7">

For some reason, everytime I request random numbers, my requests keep failing. 
I have been stuck on this for 2 days now, I am adding the conusmer and requesting random numbers by setting the requestRandomNumbers function inside the smart contract to true but it keeps failing.

The contract balance is 0.06 ETH, and it requires a max of 0.03 ETH for the transaction to take place. What could be the cause here ?

**Comment by EngrPips** [comment]
Hello @LordAdhaar, Please fund your subscription with 45 Link and retry.

**Reply by LordAdhaar** [reply]
Ok boss, @EngrPips to the rescue

**Reply by EngrPips** [reply]
Anytime, Friend. Let us know how it plays out.

**Comment by LordAdhaar** [comment]
Hey bro,
I set requestRandomWords to false and used LINK.
IT WORKED.

Is there any way for me to connect with you and ask you a few questions regarding web3 and making a career in it. You have helped me countless times now. I respect you and the help you provide to noobs like me ser.

**Reply by EngrPips** [reply]
I am @Engrpips on both discord and Twitter. Looking forward to hearing from you. Friend.
